CHANGELOG.md

```diff
@@ -10,10 +10,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 
+## [0.25.5] - 2024-08-06
```

Comment:
We aren't releasing the version in this PR, so it can remain under "unreleased" :) 

```suggestion
```

Reply:
oki